README.md

```diff
@@ -19,3 +20,6 @@ Please see the [end user documentation](end_users) for more information.
 ## Developers
 
 Please see the [developer documentation](developers) for more information.
+
+## Administrators
```

Comment:
Per our [`markdowdlint` rules](https://github.com/BonnyCI/lore/blob/master/.markdownlint.js), please add a newline after this header:
```markdown
## Administrators

Please see the [admin documentation](admins) for more information.
```